src/main/java/com/w6/nlp/ViolentVerbsParser.java

```
@@ -0,0 +1,89 @@
+package com.w6.nlp;
+
+import edu.stanford.nlp.ling.CoreLabel;
+import edu.stanford.nlp.ling.WordTag;
+import edu.stanford.nlp.parser.lexparser.LexicalizedParser;
+import edu.stanford.nlp.process.CoreLabelTokenFactory;
+import edu.stanford.nlp.process.Morphology;
+import edu.stanford.nlp.process.PTBTokenizer;
+import edu.stanford.nlp.process.Tokenizer;
+import edu.stanford.nlp.process.TokenizerFactory;
+import edu.stanford.nlp.trees.Tree;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.StringReader;
+
+import java.util.*;
+
+public class ViolentVerbsParser {
+    
+   Set violentWords;
+   String globalPpath = "./src/main/resources/violentVerbsDictionary.txt";
+   final String PCG_MODEL = "edu/stanford/nlp/models/lexparser/englishPCFG.ser.gz";
+   TokenizerFactory<CoreLabel> tokenizerFactory;
+   LexicalizedParser parser;
+   
+   
+   public ViolentVerbsParser(){
+       tokenizerFactory = PTBTokenizer.factory(new CoreLabelTokenFactory(),
+                        "invertible=true");
+       parser = LexicalizedParser.loadModel(PCG_MODEL);
+       try {
+           setViolentDictionary();
+       } catch (Exception e) {
+       }
+   }
+   
+   private void setViolentDictionary() throws IOException{
+       
+        violentWords = new HashSet<String>();
+        
+        try{
+            BufferedReader br = new BufferedReader(new FileReader(globalPpath));
+            String str;
+            while ((str = br.readLine()) != null) {
+                str = str.toLowerCase();
+                str.replaceAll("\\s+","");
+                violentWords.add(str);
+            }
+        } catch (IOException e){}
```

Don't ignore exceptions. If you can not read you can not proceed with a code.